web/src/common/entryType.enum.ts

```diff
@@ -3,4 +3,6 @@ export enum EntryType {
   FlexLeave = 101,
   SickLeave = 109,
   Vacation = 116,
+  holidayPayLeave = 139,
+  travelTime = 141,
```

Comment:
Very minor issue, but still, style should be kept consistent, so these two additions should be capitalized like the rest of the enum keys -> `HolidayPayLeave` and `TravelTime`.

Other than that, looks very good! 💪